external-builds/pytorch/setup_venv.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Set VENV_DIR from argument or default
+VENV_DIR="${1:-$(realpath "$(dirname "${BASH_SOURCE[0]}")/../..")/venv}"
+
+# Validate INDEX_URL
+if [ -z "$INDEX_URL" ]; then
+  echo "Error: INDEX_URL is not set" >&2
+  exit 1
+fi
+
+# Validate requirements-test.txt
+REQUIREMENTS_FILE="${VENV_DIR%/*}/requirements-test.txt"
+if [ ! -f "$REQUIREMENTS_FILE" ]; then
+  echo "Error: $REQUIREMENTS_FILE does not exist" >&2
+  exit 1
+fi
+
```

Comment:
This check is confusing (why use `VENV_DIR` in the path to the requirements file at all?). I would delete this whole block and any on the `pip install -r requirements-tests.txt` command failing on the file not existing instead.

```suggestion
```